drivers/sdp3x/sdp3x.c

```diff
@@ -0,0 +1,473 @@
+                                     uint8_t clkStretching)
+{
+    _SDP3x_start_triggered(dev, tempComp, clkStretching);
+#ifndef SDP3X_PARAM_IRQ_PIN
+    /* Wait for measurement to be ready if irq pin not used */
+    xtimer_usleep(46000);
+#else
+    uint32_t time_now = xtimer_now_usec();
+    /* If for some reason the pin values are not as expected */
+    if (get_irq_pin_value(dev) == 0) {
+        xtimer_usleep(46000);
+    }
+    /* Check for change in IRQn pin to signal data is ready */
+    else {
+        while (get_irq_pin_value(dev) != 0 &&
+               (xtimer_now_usec() - time_now) < 46000) {}
+    }
+#endif
+    double temp = _SDP3x_read_temp(dev);
+    return temp;
+}
+
+double SDP3x_read_single_differential_pressure(const sdp3x_t *dev,
+                                               uint8_t tempComp,
+                                               uint8_t clkStretching)
+{
+    _SDP3x_start_triggered(dev, tempComp, clkStretching);
+#ifndef SDP3X_PARAM_IRQ_PIN
+    /* Wait for measurement to be ready if irq pin not used */
+    xtimer_usleep(46000);
+#else
+    uint32_t time_now = xtimer_now_usec();
+    /* If for some reason the pin values are not as expected */
+    if (get_irq_pin_value(dev) == 0) {
+        xtimer_usleep(46000);
+    }
+    /* Check for change in IRQn pin to signal data is ready */
+    else {
+        while (get_irq_pin_value(dev) != 0 &&
+               (xtimer_now_usec() - time_now) < 46000) {}
+    }
+#endif
+    double temp = _SDP3x_read_pressure(dev);
+    return temp;
+}
+
+int8_t SDP3x_read_single_measurement(const sdp3x_t *dev, uint8_t tempComp,
+                                     uint8_t clkStretching,
+                                     sdp3x_measurement_t *result)
+{
+    _SDP3x_start_triggered(dev, tempComp, clkStretching);
+#ifndef SDP3X_PARAM_IRQ_PIN
+    /* Wait for measurement to be ready if irq pin not used */
+    xtimer_usleep(46000);
+#else
+    uint32_t time_now = xtimer_now_usec();
+    /* If for some reason the pin values are not as expected */
+    if (get_irq_pin_value(dev) == 0) {
+        xtimer_usleep(46000);
+    }
+    /* Check for change in IRQn pin to signal data is ready */
+    else {
+        while (get_irq_pin_value(dev) != 0 &&
+               (xtimer_now_usec() - time_now) < 46000) {}
+    }
+#endif
+    /* read in sensor values here */
+    int16_t data[3];
+    uint8_t ret = _SDP3x_read_data(dev, data);
+    if (ret != 0) {
+        DEBUG("[SDP3x] Error reading date\n");
+        return 0;
+    }
+    else {
+        result->differential_pressure = _SDP3x_convert_to_pascal(dev, data[0]);
+        result->temperature = _SDP3x_convert_to_celsius(data[1]);
+        result->dp_scale_factor = data[2];
+    }
+    return 1;
+}
+
+int8_t  SDP3x_start_continuous(const sdp3x_t *dev, uint8_t tempComp,
+                               uint8_t avg, uint32_t interval,
+                               sdp3x_measurement_t *result)
+{
+    (void)interval;
+    (void)result;
```

Comment:
If those are not used, why keep them in the function definition?

Reply:
Resolved in fixup commit 6b544ab8d1ee03eeff471b3d4190e00be1f6e85f